daisy_workflows/image_import/inspection/tests/test_model.py

```diff
@@ -12,56 +12,56 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Tests functions exposed by the inspection.model module."""
+"""Tests functions exposed by the di.model module."""
 
 import json
 import unittest
 
-import model
+from boot_inspect import model
 
 
-class TestDistro(unittest.TestCase):
+class TestDistro:
 
   def test_name_lookup_is_case_insensitive(self):
-    self.assertEqual(model.Distro.UBUNTU, model.distro_for("ubuntu"))
-    self.assertEqual(model.Distro.UBUNTU, model.distro_for("Ubuntu"))
-    self.assertEqual(model.Distro.OPENSUSE, model.distro_for("openSUSE"))
-    self.assertEqual(model.Distro.RHEL, model.distro_for("RHEL"))
-    self.assertEqual(model.Distro.CENTOS, model.distro_for("CentOS"))
-    self.assertEqual(model.Distro.CENTOS, model.distro_for("centos"))
+    assert model.Distro.UBUNTU == model.distro_for("ubuntu")
+    assert model.Distro.UBUNTU == model.distro_for("Ubuntu")
+    assert model.Distro.OPENSUSE == model.distro_for("openSUSE")
+    assert model.Distro.RHEL == model.distro_for("RHEL")
+    assert model.Distro.CENTOS == model.distro_for("CentOS")
+    assert model.Distro.CENTOS == model.distro_for("centos")
 
   def test_name_lookup_returns_None_when_no_match(self):
-    self.assertIsNone(model.distro_for("not-a-distro-name"))
-    self.assertIsNone(model.distro_for(""))
+    assert model.distro_for("not-a-distro-name") is None
+    assert model.distro_for("") is None
 
 
 class TestVersion(unittest.TestCase):
 
   def test_split_happy_case(self):
-    self.assertEqual(model.Version("14", "04"), model.Version.split("14.04"))
-    self.assertEqual(model.Version("2008", "3"), model.Version.split("2008.3"))
-    self.assertEqual(model.Version("15", ""), model.Version.split("15"))
+    assert model.Version("14", "04") == model.Version.split("14.04")
+    assert model.Version("2008", "3") == model.Version.split("2008.3")
+    assert model.Version("15", "") == model.Version.split("15")
 
   def test_split_empty_input(self):
-    self.assertEqual(model.Version("", ""), model.Version.split(""))
+    assert model.Version("", "") == model.Version.split("")
 
   def test_split_allows_non_period(self):
-    self.assertEqual(
-      model.Version("fuzzy/fossa", ""), model.Version.split("fuzzy/fossa"))
-    self.assertEqual("fuzzy/fossa", str(model.Version.split("fuzzy/fossa")))
+    assert model.Version("fuzzy/fossa", "") == \
+           model.Version.split("fuzzy/fossa")
+    assert "fuzzy/fossa" == str(model.Version.split("fuzzy/fossa"))
 
   def test_repr_happy_case(self):
```

Comment:
What is repr? It's not obvious what is being tested here. I assume stringification of model.Version? A better name for these tests is a probably good idea.

Reply:
yeah, it's the python method for implementing "to string". Updated the name to use "to_string" instead of "repr".